### ISSUE TITLE
Support for software serial/uart on ARM platforms (DUE) using soft_uart 

### ISSUE THREAD

**Issue by oysteinkrog** [teemuatlut/TMCStepper#23]
Has anyone looked into adding support for software uart on the DUE, using the https://github.com/antodom/soft_uart library?

**Comment by teemuatlut** [comment]
Yes I'm aware of it but it has a bit different API for setting it up and so requires additional work to get working.
